server/plugin.go

```diff
@@ -83,7 +83,7 @@ type externalConfig struct {
 	DisplaySubscriptionNameInNotifications bool
 }
 
-const defaultMaxAttachmentSize = utils.ByteSize(10 * 1024 * 1024) // 10Mb
+const defaultMaxAttachmentSize = utils.ByteSize(100 * 1024 * 1024) // 100Mb
```

Comment:
I'm not understanding why we would need a `defaultMaxAttachmentSize` in the Jira plugin.  The value is either the MM server `MaxFileSize` or the value specified in the plugin config.  It would never be this default.